tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -100,8 +101,10 @@ def misc_balances(spec):
     Usage: `@with_custom_state(balances_fn=misc_balances, ...)`
     """
     num_validators = spec.SLOTS_PER_EPOCH * 8
-    num_misc_validators = spec.SLOTS_PER_EPOCH
-    return [spec.MAX_EFFECTIVE_BALANCE] * num_validators + [spec.MIN_DEPOSIT_AMOUNT] * num_misc_validators
+    balances = [spec.MAX_EFFECTIVE_BALANCE * 2 * i // num_validators for i in range(num_validators)]
```

Comment:
2x is to have a substantial number above the MAX_EFFECTIVE?

Reply:
yes, the middle of the linear distribution being the max-effective balance. Lots of different distributions work, open for suggestions.